portal/eproms_substudy_tailored_content/webpack.config.js

```diff
@@ -0,0 +1,194 @@
+        alias: {
+          'jquery': path.join(__dirname, '/node_modules/jquery/dist/jquery.min.js')
+        }
+    },
+    devtool: 'inline-cheap-module-source-map',
+    module: {
+      rules: [
+        {
+          test: require.resolve('jquery'),
+          loader: 'expose-loader',
+          options: {
+            exposes: ['$', 'jQuery!jquery']
+          },
+        },
+        {
+          test: /\.(woff2|eot|ttf|woff|otf)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+          //use: 'url-loader?limit=10000',
+          loader: "file-loader",
+          options: {
+            name: "[name].[ext]"
+          }
+        },
+        {
+          test: /\.(ttf|svg|png|jpe?g|gif)(\?[\s\S]+)?$/,
+          loader: 'file-loader',
+          options: {
+            name: "[name].[ext]"
+          }
+        },
+        {
+          test: /\.(js)$/,
+          exclude: [/node_modules/, path.resolve(__dirname, vendorPath)],
+          loader: 'babel-loader',
+          query: {
+            presets: [
+              ['@babel/preset-env'],
+            ],
+          },
+        },
+        {
+          test: /\.css$/,
+          use: [
+            {
+              loader: MiniCssExtractPlugin.loader
+            },
+            'css-loader',
+          ],
+        },
+        {
+          test: /\.less$/,
+          use: [
+            {
+              loader: 'style-loader', // creates style nodes from JS strings
+            },
+            {
+              loader: 'css-loader', // translates CSS into CommonJS
+              options: {
+                sourceMap: isDevelopment ? true: false
+              }
+            },
+            {
+              loader: 'less-loader', // compiles Less to CSS
+              options: {
+                sourceMap: isDevelopment ? true: false
+              },
+            },
+          ],
+        },
+        {
+          test: /\.vue$/,
+          use: 'vue-loader'
+        }
+      ],
+    },
+    plugins: [
+      new VueLoaderPlugin(),
+      new HtmlWebpackPlugin({
+        title: "Substudy Tailored Content",
+        template: path.join(__dirname, '/src/app.html'),
+        filename: path.join(__dirname, `${templateDirectory}/substudy_tailored_content.html`),
+        favicon: path.join(__dirname, '/src/assets/favicon.ico'),
+      }),
+      new webpack.ProvidePlugin(
+        { 
+          Promise: 'es6-promise',
+          $: 'jquery',
+          jquery: 'jquery',
+          jQuery: 'jquery',
+          'window.jQuery': 'jquery',
+          Vue: ['vue/dist/vue.esm.js', 'default'],
+        }
+      ),
+      new MiniCssExtractPlugin({
+        // Options similar to the same options in webpackOptions.output
+        // both options are optional
+        filename: '[name].[contenthash].css',
+        chunkFilename: '[name].[id].[contenthash].css',
+      }),
+      new webpack.DefinePlugin({
+        "process.env.NODE_ENV": JSON.stringify(
+          isProduction ? "production" : "development"
```

Comment:
why not just a boolean?  seems like clients might forget and just check the truthiness of `isProduction`?

Reply:
thank you, fixed